packages/core/tests/controllers/NetworkController.test.ts

```diff
@@ -41,6 +46,7 @@ describe('NetworkController', () => {
   it('should update state correctly on switchCaipNetwork()', async () => {
     await NetworkController.switchActiveNetwork(caipNetwork)
     expect(NetworkController.state.caipNetwork).toEqual(caipNetwork)
+    expect(EventsController.state.data).toEqual(switchNetworkEvent)
```

Comment:
@arein added this to the test, but I'm not sure if you intended the test-suite to be encapsulated for each controller. 
In that case I think  it should be done in some kind of cross-controller test / something more e2e-ish right?